### DISCUSSION TITLE
C++ how to read request body

### DISCUSSION THREAD

**Discussion by JSH32** [uNetworking/uWebSockets#1127] [Q&A]
There is no body parameter or documentation on how to read the body

**Comment by JSH32** [comment]
I have figured out it is with onData but the issue now is that while reading data the connection aborts due to taking too long. Also segfaults in the process since im doing stuff in the request.

**Comment by ghost** [comment]
There are several examples of how to read a body in the JS repo.

**Comment by JSH32** [comment]
I use C++. It segfaults and fails when it takes too long. I am using REST not sockets

**Comment by ghost** [comment]
So fix your app? It's not a bug in the library.
